### PR TITLE
Show a message with the location of the listening url

### PR DESCRIPTION
**Pull request by minhajuddin**

### CODE REVIEW

**Comment by minhajuddin** [comment]
Potential use case: https://news.ycombinator.com/item?id=13021319

File: shinatra.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+echo "starting server at: http://localhost:8080/"
```
Comment (nir0s) [inline shinatra.sh]: Um, this should probably be:

```bash
PORT="${1:-8080}"
echo "Starting server at: http://localhost:${PORT}/"
```

Comment (nir0s) [inline shinatra.sh]: and then also change the variable in line 4

**Comment by diamondo25** [comment]
Note that README.md should be changed, because you introduced a new line, making the file 6 lines long

**Comment by nir0s** [comment]
True, true..

**Comment by benrady** [comment]
I can't accept any pull requests that compromise the artistic integrity of this project (as defined by the README)

**Pull request closed without merging**